flink-runtime/src/test/java/org/apache/flink/runtime/scheduler/adaptivebatch/DefaultVertexParallelismAndInputInfosDeciderTest.java

```diff
@@ -542,6 +542,12 @@ public long getNumBytesProduced() {
             return producedBytes;
         }
 
+        @Override
+        public long getNumBytesProduced(
+                IndexRange partitionIndexRange, IndexRange subpartitionIndexRange) {
+            return producedBytes;
```

Comment:
It's better to throw an `UnsupportedException` instead of returning a wrong value.